packages/office-ui-fabric-react/src/components/Panel/examples/Panel.LightDismiss.Example.tsx

```diff
@@ -15,17 +16,24 @@ export class PanelLightDismissExample extends React.Component<any, any> {
       <div>
         <DefaultButton
           text='Open panel'
-          onClick={ () => this.setState({ showPanel: true }) }
+          onClick={ this._setShowPanel(true) }
```

Comment:
Split this into 2 separate autobound functions and pass them in.
```tsx
onClick={ this._onShowPanel }
onClick={ this._onHidePanel }
```

Reply:
Or, alternatively, just suppress and leave as was, for examples. The purpose of the remove lambdas tslint rule is to encourage components to always have immutable input, where lamdas create mutations on every render. This change is really just working around the spirit of the rule and is adding unnecessary overhead, so I'm ok just suppressing.